### DISCUSSION TITLE
Open AI CLIP

### DISCUSSION THREAD

**Discussion by prashan** [allenai/allennlp#5660] [Guides]
How can I add OpenAI CLIP embeddings(vision, text) to Allennlp?

**Comment by dirkgr** [comment]
There isn't just one simple component you can use to do that, but [the OpenAI Clip API seems quite accessible](https://github.com/openai/CLIP#api). If you don't need to fine-tune CLIP itself, I would recommend calling the CLIP API inside your dataset reader, and making the resulting tensors part of your instances.
